extensions/resteasy-reactive/rest/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/customproviders/MultiResponseHeadersTest.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.resteasy.reactive.server.test.customproviders;
```

Comment:
shall I delete this in favor of `ResponseHeaderTest` cases?

Reply:
It probably makes sense to merge them, yeah

Reply:
removed, all cases are covered by ResponseHeaderTest